src/containers/greeting/Greeting.js

```diff
@@ -52,7 +52,9 @@ export default function Greeting() {
           </div>
           <div className="greeting-image-div">
             {illustration.animated ? (
-              <DisplayLottie animationData={landingPerson} />
+              <div className="fade-in">
```

Comment:
We could use `<Fade bottom duration={1000} distance="40px">` from line 19 so both the text and the lottie appears with the same effect together

Reply:
Could you check how that looks and if they fade in from bottom together?

Reply:
A new commit would also trigger a redeploy

Reply:
Sure! I'll take a look and get back to you!